src/components/Rockets/Rockets.jsx

```diff
@@ -0,0 +1,34 @@
+import { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import Rocket from './Rocket';
+import { fetchRockets } from '../../redux/rockets/rockets';
```

Comment:
- [ ] Nice work done on your Rockets component, kindly check this import `../../redux/rockets/rockets, I had issues starting your project because the name of the folder starts with a capital `R`edux and you imported it using a Lowercase `r`edux